### ISSUE TITLE
New line and tab characters get removed when '\' is used to split a line in heredoc

### ISSUE THREAD

**Issue by ssumit3** [etingof/apacheconfig#83]
If we use '\\' to split a line inside a heredoc then all the new line and tab characters are getting removed. 
For example:

```
PYTHON <<END
def fn():
	print "hi"
	return 1 + fn2()

def fn2():
	return 3
END

```
produces:

`{'PYTHON': 'def fn():\n\tprint "hi"\n\treturn 1 + fn2()\n\ndef fn2():\n\treturn 3'}`

where as 

```
PYTHON2 <<END2
def fn():
        print "hi"
        return 1 + \
	fn2()

def fn2():
        return 3

END2

```
produces:

`{'PYTHON2': 'def fn(): print "hi" return 1 + fn2() def fn2(): return 3'}`

**Comment by etingof** [comment]
Cc: @sydneyli

**Comment by sydneyli** [comment]
Weird, I can't reproduce this. I figured we should add tests along these lines anyways in #84 
@ssumit3 are you using any options to `apacheconfig`, and which version are you on?

**Comment by ssumit3** [comment]
Hi @sydneyli , I am using the following options :

    options = {
        'noescape': True,
        'autotrue': True,
        'ccomments': False
    }

Also I am version "0.2.9" with changes last pulled on 10/9/2019

**Comment by sydneyli** [comment]
Got it! Pushed a fix suggestion in #83. Thanks for reporting.
